<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConvertFile</class>
 <widget class="QDialog" name="DialogConvertFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Files</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonInputFile">
        <property name="text">
         <string>Input File...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditPath" name="lineEditOutputFile"/>
      </item>
      <item row="0" column="1">
       <widget class="LineEditPath" name="lineEditInputFile"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonOutputFile">
        <property name="text">
         <string>OutputFile...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Log</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonConvert">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Convert file and leave dialog open&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Convert</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>316</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditPath</class>
   <extends>QLineEdit</extends>
   <header>LineEditPath.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonInputFile</tabstop>
  <tabstop>lineEditInputFile</tabstop>
  <tabstop>pushButtonOutputFile</tabstop>
  <tabstop>lineEditOutputFile</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
